src/core/include/openvino/core/model.hpp

```diff
@@ -29,6 +30,16 @@ class Model;
 OPENVINO_API
 std::shared_ptr<Model> clone_model(const Model& func, std::unordered_map<Node*, std::shared_ptr<Node>>& node_map);
 
+const ov::Any& get_rt_info(const ov::Model& model,
```

Comment:
Why is it `get_tf_info`? It looks very awkward because it is reduced and run-time info is not a common term?  Why not just using `get_meta_info` etc.?

Reply:
Because we don't have meta information in new IR

Reply:
No, I mean that we select the strange name `get_rt_info` that is weird name in API? Who will use it?

Reply:
@rkazants `rt_info` will be a section in the new IRs.
And all this information will be collected inside the rt_info of `ov::Model`. It is why we have such good name for the method.